readthedocs/templates/core/project_details.html

```diff
@@ -66,7 +66,7 @@ <h3>{% trans "Build a version" %}</h3>
           <option value="{{ version.slug }}">{{ version.slug }}</option>
         {% endfor %}
         </select>
-        <input type="submit" value="{% trans "Build" %}">
+        <input type="submit" value="{% trans "Build Project" %}">
```

Comment:
As this is a build version form, perhaps this should be `Build version` instead

Reply:
Oh! Yes. You are right. I'll change it right away!